.github/workflows/publish-content-push-main.yml

```diff
@@ -50,3 +50,11 @@ jobs:
            export DEPLOY_DOMAIN=https://che-plugin-registry-main.surge.sh
            echo "DEPLOY_DOMAIN=$DEPLOY_DOMAIN" >> $GITHUB_ENV
            surge ./content --domain $DEPLOY_DOMAIN
+      - name: Publish to gh-pages
+        env:
+          GITHUB_TOKEN: ${{ secrets.CHE_BOT_GITHUB_TOKEN }}
+          VERSION_DIR: nightly
```

Comment:
is it nightly as we push on every commit ?

Reply:
it is taking the nightly tag of the docker image, isn't it ?

Reply:
it's taking image being build after each commit (image may be named nightly which is wrong at it's updated at every commit)
so `current`, `next`, `main` (like main branch) would be better name

Reply:
Updated using main